### PR TITLE
Improved exchange form

### PR DESCRIPTION
**Pull request by iower**
## Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/swaponline/swap.react/wiki/CONTRIBUTING) guide
- [x] Styleguide check `npm run validate`
- [x] Good naming, keep simple
- [x] Tested desktop/mobile
- [x] Tested bright/dark
- [x] Tested en/ru
- [x] Affects money; I checked the functionality once again
- [x] I checked the PR once again

### Original issue

#3284

### Video / screenshot proof

image

### CODE REVIEW

File: src/front/shared/pages/Exchange/Exchange.js [hunk 1]
```diff
@@ -141,22 +143,6 @@ export default class Exchange extends Component {
     orders: [],
   };
 
-  isPeerBanned(peerID) {
```
Comment (shendel) [inline src/front/shared/pages/Exchange/Exchange.js:144]: это надо оставить

Comment (shendel) [reply src/front/shared/pages/Exchange/Exchange.js:144]: https://github.com/swaponline/MultiCurrencyWallet/issues/1804

**Review by shendel — COMMENTED** [review]
-

File: src/front/shared/pages/Exchange/Exchange.js (hunk 1 above)
Comment (iower) [reply src/front/shared/pages/Exchange/Exchange.js:144]: Just diff misleading
image

**Pull request merged**